File: app/src/main/java/com/song/deviceinfo/ui/input/InputViewModel.java
package com.song.deviceinfo.ui.input;

import com.song.deviceinfo.info.InputInfo;
import com.song.deviceinfo.model.beans.InputBean;
import com.song.deviceinfo.ui.base.BaseViewModel;

import java.util.List;

/**
 * Created by chensongsong on 2020/6/4.
 */
public class InputViewModel extends BaseViewModel {

    protected List<InputBean> getInputInfo() {
        return InputInfo.getInputInfo();
    }

}

File: app/src/main/java/com/song/deviceinfo/ui/input/InputViewModel.kt
package com.song.deviceinfo.ui.input

import com.song.deviceinfo.info.InputInfo
import com.song.deviceinfo.model.beans.InputBean
import com.song.deviceinfo.ui.base.BaseViewModel

/**
 * Created by chensongsong on 2020/6/4.
 */
class InputViewModel : BaseViewModel<InputBean>() {
    val inputInfo: List<InputBean>
        get() = InputInfo.inputInfo
}
